napi-inl.h

```diff
@@ -277,7 +277,7 @@ template <typename ContextType, typename DataType, typename CallJs, CallJs call>
 typename std::enable_if<call == nullptr>::type static inline CallJsWrapper(
     napi_env env, napi_value jsCallback, void* /*context*/, void* /*data*/) {
   if (jsCallback != nullptr) {
-    Function(env, jsCallback).Call(0, nullptr);
+    Function(env, jsCallback).Call(0, static_cast<const napi_value*>(nullptr));
```

Comment:
Why is this change needed?

Reply:
The compiler flagged it as an ambiguous case with the new overload, indicating the nullptr's type fixed the error.

Reply:
Does this mean that user might need to change their code if they are already using this pattern?